java/google/registry/config/ConfigModule.java

```diff
@@ -784,4 +784,14 @@ public static int provideMaxChecks(RegistryConfig config) {
   public static Duration getAsyncDeleteFlowMapreduceDelay() {
     return Duration.standardSeconds(90);
   }
+
+  /**
+   * Per RFC {@link: https://tools.ietf.org/html/rfc5730} 'svID' is the element that
+   * contains the name of the server
```

Comment:
Just to be clear, I was mostly suggesting a better name for the actual @Config qualifier (i.e. make it @Config("greetingServerId").  The method name should probably match the @Config name, whatever it is, but it's less important since the method is never actually referenced by other user-visible code, unlike the @Config annotation value which is.